wiki/PlaceholderExpansion.md

```diff
@@ -1,397 +1,289 @@
-## Overview
-This page covers how you can use the `PlaceholderExpansion` to add own placeholders to PlaceholderAPI, which then can be used by other plugins.
+[placeholderexpansion]: https://github.com/PlaceholderAPI/PlaceholderAPI/blob/master/src/main/java/me/clip/placeholderapi/expansion/PlaceholderExpansion.java
+
+[playerexpansion]: https://github.com/PlaceholderAPI/Player-Expansion
+[serverexpansion]: https://github.com/PlaceholderAPI/Server-Expansion
+[mathexpansion]: https://github.com/Andre601/Math-expansion
+
+[relational]: https://github.com/PlaceholderAPI/PlaceholderAPI/blob/master/src/main/java/me/clip/placeholderapi/expansion/Relational.java
 
-PlaceholderAPI is using Expansions for its placeholders, with PlaceholderAPI providing the core. Users can download Expansions from the cloud server using commands in-game or by going [here](https://api.extendedclip.com/home/) if they want to use the placeholder.
+## Overview
+This page will cover how you can create your own [`PlaceholderExpansion`][placeholderexpansion] which you can either [[Upload to the eCloud|Expansion cloud]] or integrate into your own plugin.
 
-## Note
-You can either make a separate jar file, to upload it to the expansion-cloud (recommended) or have it as a local class inside your plugin.
+Something to note here is, that PlaceholderAPI relies on Expansions to be installed. PlaceholderAPI is acting as the core while the Expansions are what allows other plugins to use placeholders in various messages.  
+You can download Expansions either directly from the eCloud yourself, or download them through the [[download command of PlaceholderAPI|Commands#papi-ecloud-download]].
 
-## Examples
-There are multiple methods and ways you can use the PlaceholderExpansion.  
-Those depend on what you want to display through the placeholders in the end.
+## Table of Contents
 
-* [Without external plugin](#without-external-plugin)
-* [With external plugin](#with-external-plugin)
-  * [Separate jar](#separate-jar)
-  * [Internal class](#internal-class)
+- [Getting started](#getting-started)
+  - [Common Parts](#common-parts)
+- [Without a Plugin](#without-a-plugin)
+- [With a Plugin (External Jar)](#with-a-plugin-external-jar)
```

Comment:
```suggestion

```

Reply:
Why should this be removed?

Reply:
Is it not a duplicate?
![](https://i.imgur.com/bd4AUpN.png)

Reply:
Can you not read?

Reply:
Oh no those look extremely similar, and I wouldn't necessarily call it an `Internal Jar` which is probably why I never looked at the parentheses lol